### PR TITLE
Improve mysql plugin config file to show multiple instances

### PR DESCRIPTION
**Pull request by udaysagar2177**

### CODE REVIEW

File: managed_config/10-mysql.conf [hunk 1]
```diff
@@ -16,14 +16,33 @@
...
+      Database "sfx_database2"
+    </Database>
+
+    <Database hostB_database1>
+      Host "10.128.8.3"
       Socket "/var/run/mysqld/mysqld.sock"
       User "root"
       Password "abcdABCD1."
-      Database "lamptest"
+      Database "sfx_database1"
```
Comment (mdubbyap) [inline managed_config/10-mysql.conf]: Is it likely both hosts would name database the same?  Is this a best practice?

File: managed_config/10-mysql.conf [hunk 2]
```diff
@@ -16,14 +16,33 @@
 # 
 #     ps auwxxx | grep mysql
 
+# You can also configure this plugin to monitor multiple mysql databases
+# located on different hosts
+
 LoadPlugin mysql
 <Plugin mysql>
-    <Database test>
-      Host "localhost"
```
Comment (mdubbyap) [inline managed_config/10-mysql.conf]: I would provide two mysql confs: one for single and one for multiple and describe their purposes at the top.

Comment (udaysagar2177) [inline managed_config/10-mysql.conf]: Both config files will differ only in the number of "Database" blocks. There is not really much difference. Customer can intuitively trim the 3 block config to 1 block if he wants to monitor only one database and I can provide such explanation on the top. I guess, adding two files will create more confusion.

If I am missing something, let me know your reason for maintaining two files.

File: managed_config/10-mysql.conf (hunk 1 above)
Comment (udaysagar2177) [inline managed_config/10-mysql.conf]: I just named them as such while updating the config and testing it. I can give other meaningful names like employees, customers etc

File: managed_config/10-mysql.conf (hunk 2 above)
Comment (mdubbyap) [inline managed_config/10-mysql.conf]: Then say that in the file under the section Config File Modifications.

File: managed_config/10-mysql.conf (hunk 1 above)
Comment (mdubbyap) [inline managed_config/10-mysql.conf]: I would make them sfx_databaseA1, A2, B1 or something. "Employees" is not meaningful.

**Comment by udaysagar2177** [comment]
Thanks to @beccatortell for clearly explaining how to modify config file. 
@mdubbyap please review it.

**Pull request merged**